pkg/master/master.go

```diff
@@ -96,10 +96,53 @@ const (
 	DefaultEtcdPathPrefix = "/registry"
 )
 
+type DatabaseClients struct {
+	Default      storage.Interface
+	Experimental storage.Interface
```

Comment:
If you key the below map with group & version, you don't need this experimental member at all.